src/types/value/contentRelationship.ts

```diff
@@ -44,4 +44,5 @@ export interface FilledContentRelationshipField<
 	slug?: string
 	isBroken?: boolean
 	data?: DataInterface
+	text?: string
```

Comment:
Could we use the keytext field type here for consistency? https://github.com/prismicio/prismic-client/blob/lg/text-in-links/src/types/value/keyText.ts

Reply:
Done ✅

Reply:
Actually, I reverted this back after discussing with @xrutayisire & @dani-mp. We intentionally want to keep this as a string and don't intend for it to behave in the same way as a key text field.